src/store/user-store.js

```diff
@@ -1,16 +1,48 @@
-import { makeAutoObservable } from "mobx";
+import { makeAutoObservable, flow } from "mobx";
 
 class UserStore {
   isLoggedIn = false;
 
+  userResponse = {};
```

Comment:
Ignore this, it was done [in the previous lesson](https://github.com/kenttk/marks-side-project/pull/26)